### PR TITLE
Reduce bottom padding on alert panel

### PR DESCRIPTION
**Pull request by jfurrow**

### CODE REVIEW

**Comment by mesosphere-web** [comment]
Reviewer: @daiyiwhale

File: src/js/components/AlertPanel.js [hunk 1]
```diff
@@ -8,6 +8,7 @@ var AlertPanel = React.createClass({
   displayName: 'AlertPanel',
 
   defaultProps: {
+    className: '',
```
Comment (pierlo-upitup) [inline src/js/components/AlertPanel.js]: shouldn the default be `null`?

**Comment by pierlo-upitup** [comment]
retest this please

File: src/js/components/AlertPanel.js (hunk 1 above)
Comment (jfurrow) [inline src/js/components/AlertPanel.js]: Removed.

**Comment by rcorral** [comment]
Works great!

**Pull request merged**